projects/angular/src/wizard/_wizard.clarity.scss

```diff
@@ -291,116 +166,34 @@
 
     .clr-wizard-stepnav-item {
       display: block;
-      // IE/OldEdge
-      border-left-style: solid;
-      @include mixins.css-var(
-        border-left-width,
-        clr-wizard-stepnav-border-size,
-        wizard-variables.$clr-wizard-stepnav-border-size,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        border-color,
-        clr-wizard-stepnav-item-border-color,
-        wizard-variables.$clr-wizard-stepnav-item-border-color,
-        variables.$clr-use-custom-properties
-      );
-
-      margin: 0 0 (-1 * variables.$clr-global-borderwidth) 0;
-      @include mixins.css-var(
-        padding,
-        clr-wizard-stepnav-item-padding,
-        variables.$clr_baselineRem_0_25 0 variables.$clr_baselineRem_0_25 variables.$clr_baselineRem_4px,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        color,
-        clr-wizard-stepnav-text,
-        wizard-variables.$clr-wizard-stepnav-text,
-        variables.$clr-use-custom-properties
-      );
-      @include mixins.css-var(
-        font-weight,
-        clr-font-weight-regular,
-        variables.$clr-font-weight-regular,
-        variables.$clr-use-custom-properties
-      );
+      border-left: wizard-variables.$clr-wizard-stepnav-border-size solid
+        wizard-variables.$clr-wizard-stepnav-item-border-color;
+
+      margin: 0 0 calc(-1 * #{tokens.$cds-alias-object-border-width-100}) 0;
+      padding: #{tokens.$cds-global-space-4} 0 #{tokens.$cds-global-space-4} #{tokens.$cds-global-space-3};
+      color: wizard-variables.$clr-wizard-stepnav-text;
 
       &:not(.disabled):hover .clr-wizard-stepnav-link {
-        @include mixins.css-var(
-          color,
-          string.unquote('clr-wizard-stepnav-text--active'),
-          wizard-variables.$clr-wizard-stepnav-text--active,
-          variables.$clr-use-custom-properties
-        );
-        @include mixins.css-var(
-          background-color,
-          clr-wizard-stepnav-link-hover-bg-color,
-          transparent,
-          variables.$clr-use-custom-properties
-        );
+        color: wizard-variables.$clr-wizard-stepnav-text--active;
+        background-color: transparent;
```

Comment:
I see it was `--clr-wizard-stepnav-link-hover-bg-color: #{tokens.$cds-alias-object-interaction-background-hover};` but now transparent?

Reply:
Corrected in commit b19c78a